node_js/metrics/crashes_checks/services/ElasticSearchService.js

```diff
@@ -0,0 +1,33 @@
+const api = require('../config/api');
+
+module.exports = {
```

Comment:
I thing we could extract this service to the outer folder, because it will be used to get metrics for transactions too and will be reused.
And I think file name should be 'services/elasticsearch' or 'services/elastic_search'